root/utility/generateFlowType.js

```diff
@@ -161,7 +161,7 @@ exports.generateFlowType = async function (
     +isEditDataTypeInfo: boolean,
   } */,
 )/*: Promise<string> */ {
-  const seenTypes = new Set();
+  const seenTypes = new Set/*:: <string> */();
```

Comment:
Why the flow /:: syntax here?

Reply:
'Cause the rest of the file is still using at (and I don't remember the reason, so didn't want to break anything).